distribution/test/itests/pom.xml

```diff
@@ -125,6 +117,61 @@
             <artifactId>javax.inject</artifactId>
         </dependency>
         <!-- End of pax exam dependencies -->
+        <dependency>
```

Comment:
Are all of these dependencies needed across all three modules?  If not then we can move the dependencies only needed by a single module into the module that needs them to reduce unnecessary dependencies leaking into unrelated modules.